### PR TITLE
Add alert banner indicating when a crash does not meet the Vision Zero program's reporting requirements

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17323
Closes https://github.com/cityofaustin/atd-data-tech/issues/18564

## Testing
**URL to test:** 
Local data model

**Steps to test:**
1. Create a temp crash record
2. Go to the details page of that crash, the crash diagram section should now have a message rendering as an alert
3. Update the private drive flag to YES, see the new banner at the top
4. Update it back to NO, watch the banner go away
5. Update the coordinates to be outside of Austin
6. The banner will come back this time with a message about being outside of AFP

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/Crash.js [hunk 1]
```diff
@@ -134,6 +136,19 @@ function Crash(props) {
...
       </Row>
+      {(isPrivateDrive !== false || isInAustinFullPurpose !== true) && (
+        <Row>
+          <Col>
+            <Alert color="warning">
+              This crash is not included in Vision Zero statistical reporting
+              because{" "}
+              {isPrivateDrive
+                ? "it occurred on a private drive"
+                : "it is located outside of Austin Full Purpose"}
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/Crash.js:147]: upper case or lower case Austin Full Purpose?

Comment (johnclary) [reply atd-vze/src/views/Crashes/Crash.js:147]: let's go with: `"it is located outside of the Austin full purpose jurisdiction"`

File: atd-vze/src/views/Crashes/CrashDiagram.js [hunk 2]
```diff
@@ -153,9 +154,9 @@ const CrashDiagram = ({ crashId, isCr3Stored, isTempRecord }) => {
       {diagramError &&
         (isTempRecord ? (
           <CardBody>
-            <p>
+            <Alert color="danger">
               Crash diagrams are not available for user-created crash records.
-            </p>
+            </Alert>
```
Comment (johnclary) [inline atd-vze/src/views/Crashes/CrashDiagram.js:159]: would you set the color of this alert to `info`, and also add the `fa-info-circle` icon to left of the alert text? And remove the period (`.`) from this alert and all other alerts? 😇  sorry for the poor scoping on this

File: atd-vze/src/views/Crashes/Crash.js [hunk 3]
```diff
@@ -134,6 +136,19 @@ function Crash(props) {
           }`}</h2>
         </Col>
       </Row>
+      {(isPrivateDrive !== false || isInAustinFullPurpose !== true) && (
+        <Row>
+          <Col>
+            <Alert color="warning">
```
Comment (johnclary) [inline atd-vze/src/views/Crashes/Crash.js:142]: would you add the `fa-exclamation-triangle` icon to the left of this alert's text?

**Review by johnclary — CHANGES_REQUESTED** [review]
works great—i am requesting a few changes to the design 🙏

File: atd-vze/src/queries/tempRecords.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { gql } from "apollo-boost";
 export const GET_TEMP_RECORDS = gql`
   query getTempRecords {
     crashes(
+      order_by: { record_locator: desc }
```
Comment (roseeichelmann) [inline atd-vze/src/queries/tempRecords.js:6]: sort the temp records table by the record locator to get most recent at the top

File: atd-vze/src/_nav.js [hunk 5]
```diff
@@ -48,21 +48,14 @@ export const navigation = roles => {
...
-          name: "Upload Non-CR3",
-          url: "/tools/upload_non_cr3",
-          icon: "icon-cloud-upload",
-        },
-        {
-          name: "Create Crash Record",
-          url: "/tools/create_crash_record",
-          icon: "icon-shield",
-        },
-      ],
```
Comment (johnclary) [inline atd-vze/src/_nav.js:65]: cool—i think it's fine that the route is changing 👍

**Review by johnclary — APPROVED** [review]
Great stuff—thanks @roseeichelmann! 🚢 🚢 🚢 🚢 🚢 🚢

File: atd-vze/src/_nav.js (hunk 5 above)
Comment (roseeichelmann) [reply atd-vze/src/_nav.js:65]: cool yeah i figured why not :shrug:

**Pull request merged**